bin/install-theos

```diff
@@ -230,6 +230,18 @@ darwin_mobile() {
 		exit 3
 	fi
 
+	# Check for coreutils (not installed by default on some jbs)
```

Comment:
Thanks a ton for looking into this! Did we also want to add apt(-get) here to ensure the later commands proceed?

Reply:
Thanks! And, yeah, good point. That should probably be wrapped in a !$LEGACY check and prompt about the lack of Swift support. Would you be comfortable adding that to this pr? If not, no worries.